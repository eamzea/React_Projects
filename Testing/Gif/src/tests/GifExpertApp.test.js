import "@testing-library/jest-dom";
import React from "react";
import { shallow } from "enzyme";
import GifExpertApp from "../GifExpertApp";

describe("Pruebas en <GifExpertApp/>", () => {
  test("Debe mostrarse correctamente", () => {
    const wrapper = shallow(<GifExpertApp />);

    expect(wrapper).toMatchSnapshot();
  });

  test("Debe de mostrar una lista", () => {
    const wrapper = shallow(<GifExpertApp />);

    console.log(wrapper.find("GifGrid"));
    // expect(wrapper.find("GifGrid")).toBe();
  });
});
