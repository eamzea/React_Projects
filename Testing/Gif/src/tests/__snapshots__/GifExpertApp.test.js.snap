// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifExpertApp/> Debe mostrarse correctamente 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <button>
    Agregar
  </button>
  <ol>
    <GifGrid
      category="Dragon Ball"
      key="Dragon Ball"
    />
  </ol>
</Fragment>
`;
